repository/model/settings.go

```diff
@@ -0,0 +1,92 @@
+package model
+
+import (
+	"encoding/json"
+	"fmt"
+
+	svcmodel "release-manager/service/model"
+)
+
+const (
+	keyOrganizationName      = "organization_name"
+	keyDefaultReleaseMessage = "default_release_message"
+	keySlack                 = "slack"
+	keyGithub                = "github"
+)
+
+type Settings struct {
+	OrganizationName      string
+	DefaultReleaseMessage string
+	Slack                 IntegrationSettings
+	Github                IntegrationSettings
+}
+
+type IntegrationSettings struct {
+	Enabled bool   `json:"enabled"`
+	Token   string `json:"token"`
+}
+
+func (c *Settings) MarshalJSON() ([]byte, error) {
+	type KeyValue struct {
+		Key   string `json:"key"`
+		Value any    `json:"value"`
+	}
+
+	return json.Marshal([]KeyValue{
+		{Key: keyOrganizationName, Value: c.OrganizationName},
+		{Key: keyDefaultReleaseMessage, Value: c.DefaultReleaseMessage},
+		{Key: keySlack, Value: c.Slack},
+		{Key: keyGithub, Value: c.Github},
+	})
+}
+
+func (c *Settings) UnmarshalJSON(data []byte) error {
```

Comment:
It's not used anywhere.

Reply:
Same as above in case of `json.Unmarshal`

```
DB.From(r.entity).
Select("*").
ExecuteWithContext(ctx, &resp)
```